tequilapi/endpoints/service.go

```diff
@@ -56,28 +52,32 @@ type serviceList []serviceInfo
 // swagger:model ServiceInfoDTO
 type serviceInfo struct {
 	// example: 6ba7b810-9dad-11d1-80b4-00c04fd430c8
-	ID       string      `json:"id"`
+	ID string `json:"id"`
+
 	Proposal proposalRes `json:"proposal"`
-	// example: Running
-	Status  string         `json:"status"`
-	Options serviceOptions `json:"options"`
```

Comment:
We are going to need to know at least `serviceType` (wireguard/openvpn/etc) - since you're removing dead-code this seems fine, but we do need this implemented.

Reply:
Registered to ticket #764

Reply:
`serviceType` exists in the `Proposal`.